Gruntfile.js

```diff
@@ -174,8 +175,10 @@ module.exports = function(grunt) {
         '!js/Mp3LameEncoder.min.js',
         '!js/libsignal-protocol-worker.js',
         '!js/components.js',
+        '!js/modules/**/*.js',
         'test/**/*.js',
         '!test/blanket_mocha.js',
+        '!test/modules/**/*.js',
```

Comment:
Both implementation and test `modules` are only linted by ESLint. For some of the partially ESLint-ed files, we can still run `jshint` and `jscs` to make it a smoother transition and have more confidence. Let me know what you think 😄